src/main/java/org/example/homework3/Task4.java

```diff
@@ -0,0 +1,17 @@
+package org.example.homework3;
+
+public class Task4 {
+
+    public static void main(String[] args) {
+
+        int a = 55; //dividend
+        int b = 9; //divisor
```

Comment:
the same, no sense to left a comment, if a variable has meaningfull name
```suggestion
        int divisor = 9;
```